artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/util/NettyWritable.java

```diff
@@ -106,7 +105,8 @@ public void put(ByteBuf payload) {
 
    @Override
    public void put(String value) {
-      nettyBuffer.writeCharSequence(value, StandardCharsets.UTF_8);
```

Comment:
utf8Bytes will make calculations to send Strings.
This is at least changing the wire. 

Basically we value performance with less UTf8 calculations.. sending pairs for strings, right? or am I missing something?


I'm worried about this also breaking string compatibiility.

Reply:
@clebertsuconic AFAIK this is happening on Core, but with AMQP the Strings are UTF-8 encoded using `WritableBuffer::put(String)`.
I'm not 100% sure of it: @gemmellr do you know for what it is used `WritableBuffer::put(String)`?